etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include npm.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}/wayland-*
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.npm
+noblacklist ${HOME}/.npmrc
+
+#include allow-nodejs.inc
+noblacklist ${PATH}/node
+noblacklist /usr/include/node
+
+noblacklist ${PATH}/bash
+noblacklist ${PATH}/dash
+noblacklist ${PATH}/sh
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+include /etc/firejail/disable-shell.inc
+include /etc/firejail/disable-write-mnt.inc
+include /etc/firejail/disable-xdg.inc
```

Comment:
```suggestion
include disable-common.inc
include disable-passwdmgr.inc
include disable-programs.inc
include disable-shell.inc
include disable-xdg.inc
```

1. Don't use paths, just use names.
2. `disable-write-mnt.inc` is redundant due to the `disable-mnt` below. Removing it makes the profile simpler.

Reply:
Oops, sorry about that.  The full paths were leftover from when I had this file in `~/.config/firejail` and I missed removing it when committing.